### PR TITLE
Error visibility

### PR DESCRIPTION
**Pull request by samerbuna**
Updated with local test values for APOLLO env vars

### CODE REVIEW

**Review by bodymindarts — CHANGES_REQUESTED** [review]
Integration tests are failing on GitHub actions

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "test-local-restart": "./scripts/delete-pods.sh && source ./scripts/export-local.sh && yarn run test-local",
     "test-local-clean": "yarn run restart-charts && yarn jest --clearCache && yarn run test-local",
     "test:unit": ". ./.envrc && LOGLEVEL=warn jest --config ./test/jest-unit.config.js --bail --verbose $TEST | yarn pino-pretty -c -l",
-    "test:integration": ". ./.envrc && LOGLEVEL=warn jest --config ./test/jest-integration.config.js --bail --runInBand --verbose $TEST | yarn pino-pretty -c -l"
+    "test:integration": ". ./.envrc && LOGLEVEL=warn jest --config ./test/jest-integration.config.js --bail --runInBand --verbose --detectOpenHandles $TEST | yarn pino-pretty -c -l"
```
Comment (krtk6160) [inline package.json:33]: Why is `--detectOpenHandles` needed?

Comment (samerbuna) [reply package.json]: Removed

File: src/graphql/admin/index.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const gqlAdminSchema = new GraphQLSchema({
   types: ERROR_INTERFACE_TYPES,
 })
 
-if (isDev) {
+if (!isProd) {
```
Comment (dolcalmi) [inline src/graphql/admin/index.ts:16]: will this work in test environment? do we have permissions to write?

File: src/servers/graphql-server.ts [hunk 3]
```diff
@@ -42,10 +43,22 @@ export const startApolloServer = async ({
   startSubscriptionServer = false,
 }): Promise<Record<string, unknown>> => {
   const app = express()
+
+  const apolloPulgins = isProd
```
Comment (dolcalmi) [inline src/servers/graphql-server.ts:47]: did you test this with credentials in your local environment? if this fail we will only find out when it is deployed to production

File: src/graphql/admin/index.ts (hunk 2 above)
Comment (bodymindarts) [reply src/graphql/admin/index.ts:16]: Testnet environment has `NODE_ENV=production` set so as to be as close to production as possible.

File: src/servers/graphql-server.ts (hunk 3 above)
Comment (bodymindarts) [reply src/servers/graphql-server.ts:47]: Should definitely verify its working with real creds locally once before deploy - I agree. Has this been checked @samerbuna ?

Comment (samerbuna) [reply src/servers/graphql-server.ts:47]: Yep. However, errors thrown in the context don't seem to trigger it 😞

**Review by dolcalmi — APPROVED** [review]

File: src/servers/graphql-server.ts (hunk 3 above)
Comment (bodymindarts) [reply src/servers/graphql-server.ts:47]: That sucks... guess we can merge this anyhow as it provides value, but still need to find another solution for the `context` errors.

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**